apps-rendering/src/client/liveblog.ts

```diff
@@ -9,12 +8,12 @@ function lastUpdatedDates(): void {
 			const isoDateTimeString = element.getAttribute('datetime');
 			const date = parse(dateParser)(isoDateTimeString);
 
-			if (date.kind === ResultKind.Ok) {
+			if (date.isOk()) {
 				element.textContent = `Updated: ${formatLocalTimeDateTz(
 					date.value,
 				)}`;
-			} else {
-				logger.warn(`Unable to parse and format date: ${date.err}`);
+			} else if (date.isErr()) {
+				logger.warn(`Unable to parse and format date: ${date.error}`);
```

Comment:
It's an interesting question. We also use `isOk`, but the property is called `value`.

One way to think about a `Result` type is as being one of two possible containers, `Ok` and `Err`, each with a value inside. In this implementation, the `isOk` and `isErr` methods are referring to the containers, whereas the `value` and `error` properties are referring to the values inside the containers.

It might be worth thinking about the consequences of making a parallel change to `isOk` and `value`, which I think in this scenario we'd probably also want to do for consistency? We would be renaming `value` to `ok`, to match the `isOk` method. However, this might cause confusion, because the value you get back from `.ok` would not be of type `Ok`, but would instead be the type of whatever the value inside an `Ok` is? Likewise, when calling `.err` on a `Result` you might expect to get back an `Err`, but you're instead getting back the type of the value inside the `Err`?

What are your thoughts? I'm not sure I articulated that particularly well 😬, happy to discuss further.